crates/bevy_render/src/shader/shader_reflect.rs

```diff
@@ -28,6 +28,22 @@ impl ShaderLayout {
                     bind_groups.push(bind_group);
                 }
 
+                let mut push_constant_ranges = Vec::new();
+                for push_constant_block in module.enumerate_push_constant_blocks(None).unwrap() {
+                    let range = push_constant_block.offset..push_constant_block.size;
+                    let mut stages = BindingShaderStage::NONE;
+                    if shader_stage.contains(ReflectShaderStageFlags::VERTEX) {
```

Comment:
This seems like it could be potentially non-exhaustive. Room for improved error handling?